docs/mast/mast_mastquery.rst

```diff
@@ -51,28 +143,43 @@ The basic MAST query function returns query results as an `~astropy.table.Table`
    Length = 77 rows
 
 
-Many mast services, specifically JWST and Catalog services, require the two principal keywords, 'columns' and 'filters',
-to list parameters. Positional services will also require right ascension and declination parameters, either in
-addition to columns and filters or on their own. For example, the cone search service only requires the 'ra' and
-'dec' parameters. Using the wrong service parameters will result in an error. Read the
-`MAST API services documentation <https://mast.stsci.edu/api/v0/_services.html>`__ for more information on valid
-service parameters.
+Cone search services only require positional parameters. Using the wrong service
+parameters will result in an error. Read the
+`MAST API services documentation <https://mast.stsci.edu/api/v0/_services.html>`__ for
+more information on valid service parameters.
 
 .. doctest-remote-data::
 
    >>> from astroquery.mast import Mast
    ...
-   >>> service = 'Mast.Caom.Cone'
-   >>> params = {'columns': "*",
-   ...           'filters': {}}
-   >>> observations = Mast.service_request(service, params)
+   >>> observations = Mast.mast_query('Mast.Caom.Cone', columns = 'ra')
```

Comment:
I'm a little confused about this one, why was columns changes to 'ra' from '*'? This example is meant to show how not using the right filters leads to a `TraceBackError`, so what about adding a non-coordinate filter to the cone search, since cone search expects RA/Dec and not Teff, for example?

Reply:
Columns value is defaulted to '*' when not specified so I just changed it to something users might be looking for in a cone search. And I see what you mean about the other filters, I will add that.